saleor/graphql/product/tests/queries/test_products_query_with_where.py

```diff
@@ -0,0 +1,1518 @@
+from datetime import timedelta
```

Comment:
Maybe you already added this and I missed it but would be good to include in the tests some not visible products for the customer (Like Product.is_published==False should not be returned for the customer).